sdk/storagesync/Azure.ResourceManager.StorageSync/samples/Generated/Samples/Sample_CloudEndpointResource.cs

```diff
@@ -232,10 +232,10 @@ public async Task PreRestore_CloudEndpointsPreRestore()
 
         [Test]
         [Ignore("Only validating compilation of examples")]
-        public async Task RestoreHeartbeat_CloudEndpointsRestoreheartbeat()
+        public async Task Restoreheartbeat_CloudEndpointsRestoreheartbeat()
         {
             // Generated from example definition: specification/storagesync/resource-manager/Microsoft.StorageSync/stable/2022-06-01/examples/CloudEndpoints_RestoreHeatbeat.json
-            // this example is just showing the usage of "CloudEndpoints_Restoreheartbeat" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "CloudEndpoints_restoreheartbeat" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `restoreheartbeat`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/storagesync/resource-manager/Microsoft.StorageSync/stable/2022-09-01/storagesync.json#L1736